ee/codegen/src/serializers/vellum.ts

```diff
@@ -1995,9 +1995,12 @@ export const WorkflowVersionExecConfigSerializer: ObjectSchema<
     objectSchema({
```

Comment:
can we actually remove this field? we don't use this anywhere in codegen right?

Reply:
My vote would be yes since we decided all this lives in the toml and I didn't even realize this was here.

Reply:
to be clear, I was simply advocating removing it from codegen request handling and not from the sandbox yet. I'm pretty sure we need it in the sandbox for custom images